File: nodejs/arr.js
exports.copyArray = function (src, dst, size) {
    for (let i = 0; i < size; i++) {
        dst[i] = src[i];
    }
}

exports.printArray = function (arr, size) {
    process.stdout.write("[");
    for (let i = 0; i < size; i++) {
        if (i < size - 1) {
            process.stdout.write(String(arr[i] + " "));
        } else {
            process.stdout.write(String(arr[i]));
        }
    }
    process.stdout.write("]");
}

File: python/arr.py
import sys

def copyArray(src, dst, size):
    for i in range(0, size):
        dst[i] = src[i]

def printArray(arr, size):
    sys.stdout.write("[")
    
    for i in range(0, size):
        if i < size - 1:
            sys.stdout.write(str(arr[i]) + " ")
        else:
            sys.stdout.write(str(arr[i]))

    sys.stdout.write("]")
